md/datetimes-management-tutorial.md

```diff
@@ -4,88 +4,89 @@
-1. Follow the steps to generate a contract with a COMPLEX input *employeeInput* containing a *birthdate* attribute of type **DATE ONLY** (keep the option to generate the operations automatically)
+2. In the task **Execution** tab > **Contract** section, define the contract inputs using the **Add from data...** button
+3. Select the business variable *employee*
+4. Follow the steps to generate a contract with a COMPLEX input *employeeInput* containing a *birthdate* attribute of type **DATE ONLY** (keep the option to generate the operations automatically)
 
 ### Generate the 'Update' task form
 
 1. In the task **Execution** tab, **Form** section, generate the form by clicking on the **pencil** icon
-1. Since you have a **DATE ONLY** attribute in the contract inputs, a **Date picker** widget is automatically added to the form  
-Now in the UI Designer, define a variable to retrieve the the *Employee* currently stored using the context:  
-1. In the **Variables** panel at the bottom of the editor, click on **Create a new variable**
-1. Enter the name *employee* for the new variable
-1. select the **type** *external API*
-1. Set the **API URL** of the variable to `../{{context.employee_ref.link}}`  
+2. Since you have a **DATE ONLY** attribute in the contract inputs, a **Date picker** widget is automatically added to the form  
+3. In the UI Designer, define a variable to retrieve the the *Employee* currently stored, using the context:  
+	1. In the **Variables** panel at the bottom of the editor, click on **Create a new variable**
+	2. Enter the name *employee* for the new variable
+	3. select the **type** *external API*
+	4. Set the **API URL** of the variable to `../{{context.employee_ref.link}}`  
 FYI, The value returned by the REST API for the birth date will be an ISO 8601 formatted String (e.g. "1983-01-13").  
-1. Set the **value** of the **Date picker** widget to `employee.birthdate`
-1. Update the variable *formOutput*  to change its value to :
+4. Set the **value** of the **Date picker** widget to `employee.birthdate`
+5. Update the variable *formOutput*  to change its value to :
 `return {
 	'employeeInput': $data.employee
 };`
-1. You can delete the variable *formInput*
+6. You can delete the variable *formInput*
 
 ::: info
 The Date picker widget supports the following types for its **Value** property as input:  
 * [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) String variables (with or without time information)
 * Date objects
 * Long number values (Milliseconds since epoch)  
-However, the output of the widget is always a Javascript Date object. So it will change the type of the variable bound to the widget **Value** to a Date object when the user selects a date.  
+However, to support backward compatibility, the output of the widget is always a Javascript Date object. So it will change the type of the variable bound to the widget **Value** to a Date object when the user selects a date.  
```

Comment:
"to support backward compatibility" doesn't sound good to me. I would rather say "to be backward compatible" or "to support pages designed with older UI Designer version"